test/server-test.js

```diff
@@ -13,32 +15,67 @@ describe('server', () => {
       baseUrl: 'http://localhost:9876/'
     })
   })
-  
-  after( () => {
+
+  after( function() {
     this.server.close()
   })
-  
+
   it('should exist', function(){
     assert(app)
   })
-  
-  describe('GET /', () => {
-    it('should return a 200', (done) => {
+
+  describe('GET /', function() {
+    it('should return a 200', function(done) {
       this.request.get('/', function(error, response) {
         if (error) { done(error) }
         assert.equal(response.statusCode, 200)
         done()
       })
     })
-    
-    it('should have a body with the name of the application', (done) => {
+
+    it('should have a body with the name of the application', function(done) {
       this.request.get('/', function(error, response){
         if (error) { done(error) }
         assert.include(response.body, app.locals.title)
         done()
       })
     })
-    
+
+    describe('GET /api/v1/foods', function() {
+      beforeEach(function(done) {
+        Food.createFood('carrot', 30)
+        Food.createFood('apple', 25)
+          .then(function() { done() })
+      })
+
+      afterEach(function(done) {
+        Food.emptyFoodsTable()
+          .then(function() { done() });
+      })
+
+      this.timeout(100000000)
+      it('should a list of all foods with their id, name and calories', function(done) {
+        var ourRequest = this.request
+        ourRequest.get('/api/v1/foods', function(error, response) {
+          if (error) { done(error) }
+          Food.findAll()
+            .then(function(data){
+              var id = data.rows[0].id
```

Comment:
we should aim to be consistent with "const & let" v. var for es6; let's push to refactor these! :)